src/hooks/useCookieState.ts

```diff
@@ -0,0 +1,31 @@
+import Cookies from 'js-cookie';
+import { useCallback, useState } from 'react';
+
+const useCookieState = (
```

Comment:
JSDoc comments are missing. We need to add them here :D

Reply:
So some comments are still missing. You can refer to the other hooks for guidance. Basically we need `useCookie` and also the description of what this hook does in there as well